### DISCUSSION TITLE
Dynamic title in navbar and read progress bar

### DISCUSSION THREAD

**Discussion by yuxxeun** [arthelokyo/astrowind#47] [Ideas]
how 'bout adding a dynamic header (Navbar) title when entering the blog page and replace it with a title according to the blog title that is being read and the progress bar that is running

![image](https://user-images.githubusercontent.com/50067865/197567515-a5a55112-57bb-4964-b780-5ef7f020cdeb.png)

Like this site for example: [Mozilla Mixed Reality Blog](https://blog.mozvr.com/behind-the-scenes-at-hubs-hack-week/)

**Comment by prototypa** [comment]
Hi @yuxxeun

Thanks for the feedback. This can be a good widget for the theme. I'll be working on it, thanks.

**Reply by yuxxeun** [reply]
cant waiting for that 🔥
